usage/jsgui/src/main/webapp/index.html

```diff
@@ -26,6 +26,11 @@
 
     <title>Brooklyn JS REST client</title>
 
+    <!-- TODO: following CSS to config.js -->
```

Comment:
this line means that the CSS from index.html should be minimized as well ... I will address this issue!